### DISCUSSION TITLE
Topic created via Springboot in ks8 is not listed in kafka-cluster

### DISCUSSION THREAD

**Discussion by vyom-soft** [strimzi/strimzi-kafka-operator#9129] [General]
The springboot app deployed in k8s, it creates the topic via the api. But this topic is not visible in kafka-cluster.
The springboot pod logs shows that it created topic. Got connected to the broker.
I am using strimzi-kafka 0.35 and springboot 2.7
I do no use KafkaTopic to create topic. It is crated via the java app. Where am I going wrong. 
I do not see any connection error.
Thank you.

**Comment by antonio-pedro99** [comment]
Hi @vyom-soft, 
Where's your cluster deployed?
